scripts/publishLock.sh

```diff
@@ -0,0 +1,14 @@
+#! /bin/bash
+
+BOLD="\e[1m"
+YELLOW="\e[33m"
+NO_COLOR="\033[0m"
+
+if [ ! "$IS_CI" == "true" ]; then
+    echo -e "${BOLD}${YELLOW}"
+    echo -e "============================================================================================="
+    echo -e "The CI is configured to publish when a new tag is created."
+    echo -e "If you still want to publish yourself you have to set the env variable IS_CI to \"true\""
```

Comment:
I don't think it is intuitive the name `IS_CI`. It's perhaps personal, but when I see `IS_CI = true`, the ideia that comes to me is that everything is already integrated and you shouldn't do the deploy yourself

This is _nit_, so don't take it too seriously

Reply:
That's the idea, everything is integrated, so just in case of emergencies one should do manual release

Reply:
Yeah! But in this emergencies, I think one should set `IS_CI` to false